src/coverlet.msbuild.tasks/MSBuildLogger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using ILogger = Coverlet.Core.ILogger;
+
+namespace Coverlet.MSbuild.Tasks
+{
+    class MSBuildLogger : ILogger
+    {
+        private readonly TaskLoggingHelper _log;
+
+        public MSBuildLogger(TaskLoggingHelper _log)
+        {
+            this._log = _log;
+        }
+
+        public void LogSuccess(string message)
+        {
+            LogInformation(message);
+        }
+
+        public void LogVerbose(string message)
+        {
+            _log.LogMessageFromText(message, MessageImportance.Low);
```

Comment:
maybe this(and LogInformation) should be [TaskLoggingHelper.LogMessage](https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.utilities.tasklogginghelper.logmessage?view=netframework-4.7.2) on guide [`LogMessageFromText`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.utilities.tasklogginghelper.logmessagefromtext?view=netframework-4.7.2) is used for `Logs an error message or warning from the given line of text.` but I'm not so fluent with this part of msbuild so maybe it's ok.